Core/Services/StitchCoreLocalMongoDBService/Sources/StitchCoreLocalMongoDBService/CoreLocalMongoDBService.swift

```diff
@@ -5,12 +5,11 @@ import StitchCoreSDK
 
 public final class CoreLocalMongoDBService {
     public static let shared = CoreLocalMongoDBService()
-    private var handles = 0
     private var initialized = false
-    private var _localInstances = [String: MongoClient]()
+    
+    private var _localInstances = LRUCache<String, MongoClient>(capacity: 10)
```

Comment:
[q] Why did we choose to make this an LRU cache? Are we expected to have that many threads? I think a dictionary would be more efficient for lookups, since this LRUCache does a search in linear time

Reply:
Resolved offline.